tools/backups2datalad/annex.py

```diff
@@ -100,6 +100,8 @@ async def register_url(self, key: str, url: str) -> None:
             if self.pregisterurl is None:
                 self.pregisterurl = await open_git_annex(
                     "registerurl",
+                    "-c",
+                    "annex.alwayscompact=false",
```

Comment:
I typically pass `-c` right after `git` invocation, but since annex does allow after COMMAND to overload them, I guess should be as good.  Let's proceed and see how/if works ;)